CONTRIBUTING.md

```diff
@@ -175,10 +175,67 @@ the page to activate debug mode. The state of debug mode will be remembered by
 `window.sessionStorage` and will not survive a browser restart or opening the
 docs in a new tab.
 
-### Invisible index anchors
-
-You can create index entries and invisible anchors with `X<|thing,category>`.
-To make them visible activate debug mode.
+### Search indexing rules
+
+Bits of the content can be indexed for quick navigation.
```

Comment:
Please call them "terms" or something more precise. Code can't be indexed. Also, clarify what's that "quick navigation" you talk about.